entries/addClass.xml

```diff
@@ -19,6 +19,7 @@
   <desc>Adds the specified class(es) to each element in the set of matched elements.</desc>
   <longdesc>
     <p>It's important to note that this method does not replace a class. It simply adds the class, appending it to any which may already be assigned to the elements.</p>
+    <p><code>.addClass()</code> manipulates the property of the selected elements, not the attribute. Once the property is changed, it's the browser that updates the <code>class</code> attribute accordingly. An implication of this behavior is that this method only works for documents with HTML DOM semantics (e.g., not pure XML documents).</p>
```

Comment:
@arthurvr, @AurelioDeRosa: :+1: to specifying the property. Maybe something like this?…

The <code>.addClass()</code> method manipulates the `className` *property* of the selected elements, not the `class` *attribute*. Once the property is changed, it's the browser that updates the attribute accordingly.